source/_integrations/integration.markdown

```diff
@@ -91,17 +92,19 @@ method:
   required: false
   type: string
   default: trapezoidal
+max_sub_interval:
+  description: "Applies time based integration if the source did not change for this duration. This implies that at least every `max_sub_interval` the integral is updated. Use 0 for no time based updates."
+  required: false
+  type: time
 {% endconfiguration %}
 
-In case you expect that your source sensor will provide several subsequent values that are equal, you should opt for the `left` method to get accurate readings.
-
 The unit of `source` together with `unit_prefix` and `unit_time` is used to generate a unit for the integral product (e.g. a source in `W` with prefix `k` and time `h` would result in `kWh`). Note that `unit_prefix` and `unit_time` are _also_ relevant to the Riemann sum calculation. 
 
 ## Integration method
 
-Riemann Sum is a approximation of an integral by a finite sum and is therefore intrinsically inaccurate, nonetheless, depending on the method used, values can be more or less accurate.
```

Comment:
by removing "depending on the method used, " this sentence became strange due to redundancy of "inaccurate"